packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -200,6 +221,17 @@ const getStyles = (theme: GrafanaTheme2) => {
         }
       }
     `,
+    disabled: css`
+      color: ${theme.colors.action.disabledText};
```

Comment:
I could use some help here:
Will the color alone be enough or should the background be changed as well?
<img width="257" alt="image" src="https://user-images.githubusercontent.com/3773351/203058834-7df2645f-f407-427c-a76e-7290c1fe09f7.png">
<img width="262" alt="image" src="https://user-images.githubusercontent.com/3773351/203058868-3e28f808-e988-4a94-86c6-9e1c6c646742.png">

There's also `theme.colors.action.disabledOpacity` which results into this:
<img width="256" alt="image" src="https://user-images.githubusercontent.com/3773351/203059497-4417b045-e5b6-4484-b6d3-2011dc8f32b6.png">